azure-pipelines.yml

```diff
@@ -0,0 +1,39 @@
+variables:
+  PreferredToolArchitecture: x64
+      
+jobs:
+- job: Windows
+  timeoutInMinutes: 0
+  pool:
+    vmImage: 'vs2017-win2016'
+  strategy:
+    matrix:
+      ITK4:
+        itk.version: 'v4.13.1'
+      ITK5:
+        itk.version: 'v5.0b03'
```

Comment:
In general I agree that it's nice to use tagged ITK versions. But unfortunately, with both of these two (v4.13.1 and v5.0b03), elastix still suffers from https://github.com/InsightSoftwareConsortium/ITK/issues/350 Would it be possible to pick the last versions from https://github.com/InsightSoftwareConsortium/ITK/tree/release-4.13 and ITK git master instead?

Reply:
Yes sure, done!